src/components/Pictures.js

```diff
@@ -0,0 +1,28 @@
+import React from 'react'
+import { Composition, Box } from 'atomic-layout'
+import styled from 'styled-components'
+
+const StyledImage = styled.div`
+  border: 8px solid black;
+
+  :hover {
+    transition: background-color 1.5s;
+    background-color: black;
+  }
+`
+
+function Pictures({ items }) {
+  return (
+    <div>
```

Comment:
There is no need for an extra `div`. 

`Composition` itself is a `div` by default. Please remove the parent div.